src/ducks/SANCharts/Charts.js

```diff
@@ -237,12 +238,12 @@ class Charts extends React.Component {
         const { metricAnomalyKey: anomaly } = rest
         const result = value || chartData[index]
         const eventsData = getEventsTooltipInfo(rest)
```

Comment:
for me better to move color calculation to [`getEventsTooltipInfo`](https://github.com/santiment/app/blob/master/src/ducks/SANCharts/utils.js#L15) function